rust/src/proofs/api.rs

```diff
@@ -830,6 +884,34 @@ fn empty_sector_update_decode_from(
     })
 }
 
+/// TODO: document
+#[ffi_export]
+unsafe fn empty_sector_update_decode_from_range(
+    registered_proof: RegisteredUpdateProof,
+    comm_d: &[u8; 32],
+    comm_r: &[u8; 32],
+    input_fd: libc::c_int,
+    sector_key_fd: libc::c_int,
+    output_fd: libc::c_int,
+    nodes_offset: u64,
+    num_nodes: u64,
```

Comment:
Just sanity checking this is in fr32 elements (so basically padded bytes * 32)?

Reply:
Yes, it's the number of elements, though it;s padded_bytes **divided** by 32.